src/com/exo/thomas/Exo.java

```diff
@@ -0,0 +1,113 @@
+package com.exo.thomas;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.Scanner;
+
+public class Exo {
+
+	public static void main(String[] args) {
+		LocalDate ajd = LocalDate.now();
+		Scanner clavier = new Scanner(System.in);
+
+// Récupérer la date de naissance de l'utilisateur
+		boolean validDate1 = false;
+		int age1 = 0;
+
+		while (!validDate1) {
+
+			System.out.println("Entrez votre date de naissance (jj/mm/aaaa) :");
+
+			String date1 = clavier.nextLine();
+			String[] jma1 = date1.split("[/\\-.\\s]");
```

Comment:
Tu peux faire plus simple. Il faudrait que le format de la date soit une constante que tu utilises dans le code après pour éviter de copier coller du code :
```suggestion
			String dateFormat = "dd/MM/yyyy";
                        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(dateFormat);
                        String birthdayDateUser = userOutput.nextLine();
		        LocalDate birthdayUser = LocalDate.parse(birthdayDateUser, dateFormatter);
```
Et ensuite tu peux faire ton between avec la date de naissance de l'utilisateur.